<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #2F80ED;</string>
  </property>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Montserrat';
font-style: normal;
font-weight: 700;
font-size: 45px;
line-height: 122px;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Кошельки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Montserrat';
font-style: normal;
font-weight: 700;
font-size: 15px;
line-height: 122px;
color: #FFFFFF;
text-align:center;</string>
   </property>
   <property name="text">
    <string>email</string>
   </property>
  </widget>
  <widget class="QListWidget" name="wallets_list">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>140</y>
     <width>331</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QListView::item
{
margin: 5px;
height: 30px;
color: #000;
background-color:#FFF;
border-radius:15px;
padding:0 10px;
}
QListView{
border: none;
}
QListView::item:selected
{
border:none;
}</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>560</y>
     <width>245</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	 background-color: #FFA400;
	border-radius: 20%;
	font-family: 'Montserrat';
	font-style: normal;
	font-weight: 400;
	font-size: 15px;
	color: #000000;
}
QPushButton:hover{
   background-color:#FFB32B;
}


</string>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="export_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>610</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font-family: 'Montserrat';
font-style: normal;
font-weight: 400;
font-size: 13px;
line-height: 16px;
color: #E4E4E4;
border: none;
text-align:center;
}
QPushButton:hover {
color:#fff
}</string>
   </property>
   <property name="text">
    <string>Экспорт в csv</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
